### ISSUE TITLE
Enable CI for concerto-ui

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/concerto-ui#28]
Currently Travis is disabled for the repo. Running `lerna run test` fails on the license check:
```
bash-3.2$ lerna run test
lerna run test
lerna notice cli v3.20.2
lerna info Executing command in 2 packages: "npm run test"
lerna ERR! npm run test exited 1 in '@accordproject/concerto-ui-core'
lerna ERR! npm run test stdout:

> @accordproject/concerto-ui-core@0.82.6 pretest /Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core
> npm run licchk


> @accordproject/concerto-ui-core@0.82.6 licchk /Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core
> license-check-and-add

Running using exclude exact_paths list
Running using exclude file type list

lerna ERR! npm run test stderr:
No default format specified using {"prepend":"/*","append":"*/"} as backup
✗ License not found in  /Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/babel.config.js
/Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/node_modules/license-check-and-add/index.js:173
      throw new Error('License Check failed. '+err.length+' file(s) did not have the license.');
      ^

Error: License Check failed. 1 file(s) did not have the license.
    at giveFilesLicense (/Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/node_modules/license-check-and-add/index.js:173:13)
    at Object.module.exports.run (/Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/node_modules/license-check-and-add/index.js:182:4)
    at start (/Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/node_modules/license-check-and-add/bin/server.js:11:10)
    at Object.<anonymous> (/Users/jeromesimeon/git/gold/concerto-ui/packages/concerto-ui-core/node_modules/license-check-and-add/bin/server.js:14:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
```

It would be nice to:
- [x] get testing to run again
- [x] enable travis, and fix corresponding travis configuration
- [x] add more react-mindful testing, maybe through jest?

**Comment by jeromesimeon** [comment]
Should be more or less fixed in PR #30

**Comment by jeromesimeon** [comment]
Little green and red marks now.

<img width="807" alt="Screenshot 2020-02-21 at 11 08 03 AM" src="https://user-images.githubusercontent.com/670099/75050676-7605a680-549a-11ea-8526-d10a19b06146.png">
